src/web/router.js

```diff
@@ -50,6 +50,7 @@ router.get('/gateway_accounts/:id/surcharge', auth.secured, gatewayAccounts.surc
 router.post('/gateway_accounts/:id/surcharge', auth.secured, gatewayAccounts.updateSurcharge)
 router.get('/gateway_accounts/:id/email_branding', auth.secured, gatewayAccounts.emailBranding)
 router.post('/gateway_accounts/:id/email_branding', auth.secured, gatewayAccounts.updateEmailBranding)
+router.post('/gateway_accounts/:id/moto_payments', auth.secured, gatewayAccounts.motoPayments)
```

Comment:
I think it would be good if the path was `/toggle_moto_payments` and the controller function was named `toggleMotoPayments` to make it clearer what the action is when this route is posted to.